lib/utils/starter-repo-mapping.js

```diff
@@ -14,6 +14,7 @@
 module.exports = {
     'react': "aws-samples/aws-mobile-react-sample",
     'react-native': "aws-samples/aws-mobile-react-native-starter",
+    'angular': "aws-amplify-angular-sample"
```

Comment:
You can comment or remove that until the new sample is released

Reply:
Sure, done. Also, updated the unit test accordingly.